README.md

```diff
@@ -57,6 +72,12 @@ In order to run this project you need:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Deployment
+
+You can deploy this project using Heroku, Netlify or any free hosting platform
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- In ReadMe file: I would advise you to acknowledge or give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).
- I am asking this because it is part of the project requirements